pkg/controller/utils/conditions.go

```diff
@@ -703,3 +703,13 @@ func FindClusterInstallCondition(conditions []hivev1.ClusterInstallCondition, co
 	}
 	return nil
 }
+
+// ConditionWithPositivePolarity checks if cluster deployment condition has positive polarity
+func ConditionWithPositivePolarity(conditionType hivev1.ClusterDeploymentConditionType) bool {
```

Comment:
it reads better if the check functions start with `Is` .. so `IsConditionWithPositivePolarity`

Reply:
changed